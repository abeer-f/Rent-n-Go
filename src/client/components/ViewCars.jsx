import React, { Component } from 'react';
import { Link } from "react-router-dom";
const ViewCars =  () => {
const [cars, setCars] = 
 
    return (
      
<div>
    <h1>View Cars Page</h1>
<nav
        style={{
          borderBottom: "solid 1px",
          marginLeft: "20px",
          paddingBottom: "1rem"
        }}
      >
        <Link to="/">View Cars</Link>
      </nav>
</div>
    
    );
  
}

export default ViewCars;