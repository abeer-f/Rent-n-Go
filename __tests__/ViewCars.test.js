/**
 * @jest-environment jsdom
 */


import React from 'react';
import '@testing-library/jest-dom'
import renderer from 'react-test-renderer';
import ViewCars from '../src/client/components/ViewCars';
import { MemoryRouter } from 'react-router-dom';

it('renders ViewCars', async () => {
  const data = { description: "" }
  const component = renderer.create(
    <MemoryRouter>
      <ViewCars rowData={data} />
    </MemoryRouter>,
  );
  let tree = component.toJSON();
  await expect(tree).toMatchSnapshot();
});
