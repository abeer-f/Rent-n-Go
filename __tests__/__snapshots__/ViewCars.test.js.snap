// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ViewCars 1`] = `
Array [
  <h1>
    View Cars Page
  </h1>,
  <nav
    style={
      Object {
        "borderBottom": "solid 1px",
        "color": "white",
        "marginLeft": "20px",
        "paddingBottom": "1rem",
      }
    }
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Back to Home Page
    </a>
  </nav>,
  <div>
    <table>
      <tr>
        <td>
          Vehicle Name
        </td>
        <td>
          Vehicle Description
        </td>
        <td>
          People Capacity
        </td>
        <td>
          Bag Capacity
        </td>
        <td>
          Currency Code
        </td>
        <td>
          Rate Plan Type
        </td>
        <td>
          Daily Price
        </td>
        <td>
          Total Price
        </td>
        <td>
          Car License No.
        </td>
        <td>
          Zip Code
        </td>
        <td>
          Street
        </td>
        <td>
          City
        </td>
        <td>
          Address Line 1
        </td>
      </tr>
    </table>
  </div>,
]
`;
